static/app/components/feedback/feedbackItem/useUpdateFeedback.tsx

```diff
@@ -24,20 +25,25 @@ export default function useResolveFeedback({feedbackItem}: Props) {
     return `/organizations/${organization.slug}/issues/${feedbackId}/`;
   }, [feedbackId, organization]);
 
-  const handleResolve = useCallback(async () => {
-    addLoadingMessage(t('Updating feedback...'));
-    try {
-      await api.requestPromise(url, {
-        method: 'PUT',
-        data: {status: feedbackItem.status === 'unresolved' ? 'resolved' : 'unresolved'},
-      });
-      addSuccessMessage(t('Updated feedback'));
-    } catch {
-      addErrorMessage(t('An error occurred while resolving the feedback.'));
-    }
-  }, [api, url, feedbackItem.status]);
+  const handleUpdate = useCallback(
+    async (newStatus: GroupStatus) => {
+      addLoadingMessage(t('Updating feedback...'));
+      try {
+        await api.requestPromise(url, {
+          method: 'PUT',
+          data: {
+            status: newStatus,
+          },
+        });
+        addSuccessMessage(t('Updated feedback'));
+      } catch {
+        addErrorMessage(t('An error occurred while updating the feedback.'));
+      }
+    },
+    [api, url]
+  );
```

Comment:
We could leverage `useMutation` here. Not required though, but i was looking at it because it might make it easier to make optimistic updates & make the refetch happen.

https://develop.sentry.dev/frontend/network-requests/#quick-start-1

Reply:
hmmm perhaps a refactor we can do in a different pr?

Reply:
yup, sounds good